resource-definitions/k8s-cluster-git/runtime/github-for-gitops.yaml

```
@@ -0,0 +1,29 @@
+apiVersion: entity.humanitec.io/v1b1
+kind: Definition
+metadata:
+  id: github-for-gitops
+entity:
+  name: github-for-gitops
+  driver_type: humanitec/k8s-cluster-git
+  type: k8s-cluster
+  driver_inputs:
+    values:
+      # Git repository for pushing manifests
+      url: git@github.com:example-org/gitops-repo.git
+      # Branch in the git repository, optional. If not specified, the default branch is used.
+      branch: development
+      # Path in the git repository, optional. If not specified, the root is used.
+      path: "${context.app.id}/${context.env.id}"
+      # Load Balancer, optional. Though it's not related to the git, it's used to create ingress in the target K8s cluster.
```

Is this clarification correct?
```suggestion
      # Load Balancer, optional. Though it's not related to the GitOps setup, it's used to create ingress in the target K8s cluster if such resources are part of the Resource Graph, just like with a non-GitOps cluster.
```